whole-tale/wt_versioning
# Use varying access level on version/runs methods and add tests

This PR adds and enables tests along with other project maintenance related boilerplate. I sneaked in one minor change related to how ACLs are handled. Before most of the models were forcefully loaded and a single routine `_checkAccess` was called to see if a user has *write* access to it. Instead of doing that we now load individual objects with proper and varied access level, which is a first step towards integration with our Tale sharing functionality.

### How to test?
There's not much to test here really. Optionally follow testing steps from https://github.com/whole-tale/ngx-dashboard/pull/55 and see it still works
